diagnostic_feedback/public/js/src/student/student_quiz.js

```diff
@@ -236,16 +275,20 @@ function StudentQuiz(runtime, element, initData) {
         var currentStep = currentIndex + 1;
       var isLast = (newIndex == $(studentViewFormSecSelector, element).length - 1);
 
-      //log event for loading question
-      common.publishEvent({
-        event_type: 'xblock.diagnostic_feedback.quiz.question.submit',
-        quiz_type: initData.quiz_type,
-        quiz_title: initData.quiz_title,
-        current_question: currentStep
-      });
-
-
-      return saveOrSkip(isLast, currentStep, currentIndex, newIndex);
+      var status = saveOrSkip(isLast, currentStep, currentIndex, newIndex);
+      if(status == true){
```

Comment:
@jareerahsan You can shorten this to `if (status) {`. Please double check the rest of the code to make sure there's a space after each `if`.